sentences/de/fan_HassTurnOn.yaml

```
@@ -5,6 +5,7 @@ intents:
       - sentences:
           - "(schalte | starte) (den Ventilator | (die | alle) Ventilatoren) <area> [<turn_on>]"
           - "(schalte | starte) <area> (den Ventilator | (die | alle) Ventilatoren) [<turn_on>]"
+          - "(mach | mache) <area> (den Ventilator | (die | alle) Ventilatoren) [<turn_on>]"
```

Similar to the existing sentences above:
```suggestion
          - "(mach | mache) <area> (den Ventilator | (die | alle) Ventilatoren) [<turn_on>]"
          - "(mach | mache) (den Ventilator | (die | alle) Ventilatoren) <area> [<turn_on>]"
```